ext/Logger.php

```diff
@@ -86,13 +86,13 @@ public static function getLogger()
 
     public function beforeSuite(SuiteEvent $e)
     {
-        $suite = str_replace('\\', '_', $e->getSuite()->getName());
+        $suite = str_replace('\\', '_', $e->getSuite()->getName()) . '.log';
```

Comment:
Now the name of this variable is really wrong.
Could you rename it to something like $suiteLogFile?

There is no information about suite log files in documentation of this extension.
Could you decribe them in the docblock above this class?

Reply:
@Naktibalda  I've made what you asked for. Please, comment if this ok or not =)